packages/contracts/scripts/rename-deploy-scripts.ts

```diff
@@ -39,6 +41,8 @@ const L2_ORDERED_NAMES = [
   'L1FeeVault',
   'KromaMintableERC20Factory',
   'KromaMintableERC721Factory',
+  'GovernanceTokenProxy',
+  'MintManager',
```

Comment:
Why don't we add prefix of L2 here as well?

Reply:
It's to reduce difference from OP.